src/Components/Aspire.StackExchange.Redis/Aspire.StackExchange.Redis.csproj

```diff
@@ -14,12 +14,19 @@
     <Compile Include="..\Common\ConfigurationSchemaAttributes.cs" Link="ConfigurationSchemaAttributes.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="$(VendoringDir)OpenTelemetry.Instrumentation.StackExchangeRedis\**\*.cs" LinkBase="OpenTelemetry.Instrumentation.StackExchangeRedis" />
+    <!-- Tests need access to StackExchangeRedisInstrumentationOptions which is made internal in /Vendoring -->
```

Comment:
This is an interesting/unfortuante side-effect of this change. If users need to customize/configure the SERedis instrumentation options, previously they could. Now with this change, since we are making these options internal in our own package, customers can no longer customize it. Instead, they would have to disable `Tracing` from our component, and add it themselves, with their own configuration.

I'm not sure this changes our plans, but it is something we should be aware of.

Reply:
Implementation detail, they can't change it until it's stable.

Reply:
Or we need to create a custom options class (or change this one's namespace).

Reply:
No, I don't think we need a custom options class for this. Following the above approach will work for people who hit this. The "real fix" here is to get SE Redis to use Activities intrinsically and not even have this Instrumentation library.